.php_cs.dist

```diff
@@ -0,0 +1,30 @@
+<?php
+
+$finder = new PhpCsFixer\Finder();
+$config = new PhpCsFixer\Config('json-schema', 'json-schema style guide');
+$finder->in(__DIR__);
+
+/* Based on ^2.1 of php-cs-fixer */
+$config
+    ->setRules([
+        // default
+        '@PSR2' => true,
```

Comment:
Or maybe I misunderstood and you were not referring to phpcodesniffer (which is generally called phpcs) but rather php-cs-fixer? If that is the case, I'll add it to the Travis flow. Any feedback on the ruleset? Removals, additions?